### PR TITLE
Dmitrii_Matvienko Tasks 41-50

### PR DESCRIPTION
**Pull request by MatvienkoDO**

### CODE REVIEW

File: tasks/44_time-simulation/time-simulation.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export class SimTime {
```
Comment (MatvienkoDO) [inline tasks/44_time-simulation/time-simulation.js]: @alexkond asked us to make it in 'before es6' manner. Without 'class' language construction

File: tasks/44_time-simulation/time-simulation.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function SimTime() {
+  this.realTime = 0;
+  this.simulatedTime = 0;
+  this.speed = 1;
+}
+
+SimTime.prototype.get = function() {
```
Comment (savchukm) [inline tasks/44_time-simulation/time-simulation.js]: `this.get = ...` will be enough

File: tasks/50_camel-case/camel-case.spec.js [hunk 3]
```diff
@@ -2,12 +2,12 @@
 
 xdescribe('Camel case', () => {
 
-    test("'hello case'.camelCase() should return 'HelloCase'", () => {
```
Comment (vovcyan) [inline tasks/50_camel-case/camel-case.spec.js]: Please revert this change

File: tasks/50_camel-case/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ### CamelCase
 
-Write simple .camelCase method for strings. All words must have their first letter capitalized without spaces.
```
Comment (vovcyan) [inline tasks/50_camel-case/README.md]: Revert it

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: tasks/50_camel-case/camel-case.spec.js [hunk 5]
```diff
@@ -2,12 +2,12 @@
...
-        expect('hello case'.camelCase()).toBe("HelloCase");
+    test("'hello case'.camelCase() should return 'helloCase'", () => {
+        expect('hello case'.camelCase()).toBe("helloCase");
     });
 
-    test("'camel case word'.camelCase() should return 'CamelCaseWord'", () => {
-        expect('camel case word'.camelCase()).toBe("CamelCaseWord");
+    test("'camel case word'.camelCase() should return 'camelCaseWord'", () => {
+        expect('camel case word'.camelCase()).toBe("camelCaseWord");
```
Comment (savchukm) [inline tasks/50_camel-case/camel-case.spec.js]: It is not allowed to change test suites. 
There are 2 types of camel cases: upper camel case and lower camel case. By the way, It is not a mistake

File: tasks/50_camel-case/README.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
+Write simple .camelCase method for strings. All words except first must have their first letter capitalized without spaces.
```
Comment (savchukm) [inline tasks/50_camel-case/README.md]: Revert these changes

File: tasks/50_camel-case/camel-case.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line no-extend-native
+String.prototype.camelCase = function() {
```
Comment (savchukm) [inline tasks/50_camel-case/camel-case.js:2]: Please make this function working with correct tests

**Review by savchukm — CHANGES_REQUESTED** [review]

**Review by alexkond — DISMISSED** [review]

**Review by alexkond — APPROVED** [review]

**Review by savchukm — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**